src/languages/en.json

```diff
@@ -219,5 +219,7 @@
   "no_wallet": "No wallet detected",
   "network_mismatch": "Network Mismatch",
   "connect_to_network": "Connect your wallet to",
-  "current_connected": "Your wallet is currently connected to"
+  "current_connected": "Your wallet is currently connected to",
+  "connected_successful": "Connected Successful",
```

Comment:
I am going to change this to `Connected Successfully`, but will wait to until after the translations come back.